.github/workflows/ci.yml

```diff
@@ -102,6 +102,10 @@ jobs:
       - name: Run CSI-Sanity Tests
         run: devbox run csi-sanity-test
 
+      - name: run upstream E2E Tests
+        if: github.ref_head == 'upstream-e2e-tests'
```

Comment:
This will be changed to main before merging. Currently, its set to branch name just for testing purpose.

Reply:
updated